### PR TITLE
Diladele Web Filtering Proxy 1.5

### PR DESCRIPTION
**Pull request by ra-at-diladele-com**
- [ X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25441)

### CODE REVIEW

File: manifests/d/Diladele/WebProxy/1.5.0/Diladele.WebProxy.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: Diladele.WebProxy
+PackageVersion: 1.5.0
```
Comment (ItzLevvie) [inline manifests/d/Diladele/WebProxy/1.5.0/Diladele.WebProxy.locale.en-US.yaml:3]:
```suggestion
PackageVersion: 1.5.0.0
```

File: manifests/d/Diladele/WebProxy/1.5.0/Diladele.WebProxy.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Diladele.WebProxy
+PackageVersion: 1.5.0
```
Comment (ItzLevvie) [inline manifests/d/Diladele/WebProxy/1.5.0/Diladele.WebProxy.installer.yaml:3]:
```suggestion
PackageVersion: 1.5.0.0
```

File: manifests/d/Diladele/WebProxy/1.5.0/Diladele.WebProxy.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+PackageIdentifier: Diladele.WebProxy
+PackageVersion: 1.5.0
```
Comment (ItzLevvie) [inline manifests/d/Diladele/WebProxy/1.5.0/Diladele.WebProxy.yaml:3]:
```suggestion
PackageVersion: 1.5.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
**You'll have to update the version folder as well.**

**Comment by ra-at-diladele-com** [comment]
The verifier bot failed the changes because these do not correspond to manifest. Shall I revert them? The X.X.X.X version indication you requested seems to be incorrect. It needs to be X.X.X (three digits). Are you sure your requested changes are correct?

**Comment by ItzLevvie** [comment]
> Shall I revert them?

No because you need to update the version folder from `1.5` to `1.5.0.0` to pass pipelines.

> The X.X.X.X version indication you requested seems to be incorrect. It needs to be X.X.X (three digits).

Windows Package Manager requires that you use the PackageName, Publisher & PackageVersion directly from the Control Panel for application matching.

If the application's version number is incorrect, please recompile your application and update the hash in the package.

**Comment by ra-at-diladele-com** [comment]
Thanks, I will close this pull request and update the version folder too. I will then open another pull request.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Incorrect PackageVersion - make sure the version in the manifest matches the one in Control Panel.

**Comment by ra-at-diladele-com** [comment]
We would like to keep the 1.5.0.0 that is present in the MSI and control panel as correct product version. If I change the winget yaml files to 1.5.0.0 instead of 1.5.0 as it is now - the automated manifest checking fails. What shall I specify in the yaml files to let it succeed?

I though the three digit X.X.X scheme was required by winget manifest (which is strange because MSI is based on normal 4-digit scheme). Maybe we shall use another manifest version in yaml files? I am lost and cannot understand. Thanks.

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
> 
> 
> We would like to keep the 1.5.0.0 that is present in the MSI and control panel as correct product version. If I change the winget yaml files to 1.5.0.0 instead of 1.5.0 as it is now - the automated manifest checking fails. What shall I specify in the yaml files to let it succeed?
> 
> I though the three digit X.X.X scheme was required by winget manifest (which is strange because MSI is based on normal 4-digit scheme). Maybe we shall use another manifest version in yaml files? I am lost and cannot understand. Thanks.

WinGet does not require a three digit version schema. You need to update the Version folder name. The error says the Version folder and PacakgeVersion does not match.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**